packages/neon/neon/lib/src/pages/home.dart

```diff
@@ -48,7 +46,7 @@ class _HomePageState extends State<HomePage> {
 
     _capabilitiesBloc.capabilities.listen((final result) async {
       if (result.data != null) {
-        widget.onThemeChanged(result.data!.capabilities.theming);
+        themeNotifier.value = result.data!.capabilities.theming;
```

Comment:
I'd say this should go into the bloc. No need to only run in the widget.

Reply:
good idea :)

Reply:
Could you fix this soon? Then we can merge this and afterwards fix the dependencies and flutter.

Reply:
Oh yeah one sec